README.md

```diff
@@ -50,66 +50,76 @@ The project is structured around multiple components:
 
 # Getting started
 
-## Requirements
+## Platform Requirements
 
-- `python3`
-- `python3-venv`
+- The setup requires a Gen-6 or newer Intel CPU (for Intel PT) and sufficient
+  RAM to run several VMs at once.
+- A modifed Linux host kernel is required for VM-based snapshot fuzzing with
+  Intel PT coverage feedback. This setup does not run inside a VM or container!
 
-~~~
+## Local Installation
+
+The userspace installation and fuzzing workflow has been tested for recent
+Ubuntu (>=20.04) and Debian (>=bullseye). The base installation is captured
+as an Ansible workflow which you can bootstrap using Python:
+
+~~~sh
 sudo apt-get install python3 python3-venv
+make deploy
 ~~~
 
-## Setup
+Ansible setup will ask for your root password.
+If you are using a _passwordless sudo_ setup, just skip this by pressing enter.
 
-kAFL's deployment offers the possibility of local or remote installation.
+## Remote Installation
 
-In both cases, you will find in the installation directory:
-- `.env` file: useful environment variables for your scripts
-- `.venv` Python virtual environment: where kAFL fuzzer is installed
+kAFL's deployment offers the possibility of remote installation using Ansible.
+Update the `deploy/inventory` file according to the [Ansible inventory
+guide](https://docs.ansible.com/ansible/latest/user_guide/intro_inventory.html)
+and make sure to **remove** the first line:
 
+> localhost ansible_connection=local
 
-### Local
 
-- installation directory: `<repo_root>/kafl`
+Deployment will install kAFL to `$HOME/kafl` of the target machines:
 
-Run the depployment with:
-~~~
+~~~sh
 make deploy
 ~~~
 
-You will be prompted for your root password.
-If you are using a _passwordless sudo_ setup, just skip this by pressing enter.
+Note:
+- If your nodes require a proxy setup, update the `group_vars/all.yml`.
+- Remote deployment assumes password-less ssh and sudo.
 
-### Remote
 
-- installation directory: `$HOME/kafl`
+# Next Steps
 
-You will have to update the `deploy/inventory` file to describe your nodes, according to [Ansible's inventory guide](https://docs.ansible.com/ansible/latest/user_guide/intro_inventory.html).
-Make sure to **remove** the first line:
+## Activate the Environment
 
-~~~
-localhost ansible_connection=local
+The installation is made available as shell and python environment. You can
```

Comment:
We should provide only one way to do things as much as possible to reduce user fatigue.
So I vote for `make env`, which we can always reimplement later on as we need, instead of breaking user expectations
by changing `env.sh` content or location.